### PR TITLE
RH-DOCS-11487

### PR DESCRIPTION
**Pull request by Raghav2018**
MongoDB: Remove Data with the Java Driver

It has four methods:

- one that illustrates removing all the documents based on multiple conditions. 
- one that illustrates removing one document based on multiple conditions. 
- one that illustrates removing all the documents.
- one that illustrates dropping the collection.

### CODE REVIEW

**Comment by Raghav2018** [comment]
@shannonbradshaw I have completed coding remove methods. Please review the same.

File: RemoveData/src/RemoveData.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+		{
+			MongoClientURI clientUri = new MongoClientURI(connectionString);
+			try(MongoClient client = new MongoClient(clientUri))
+			{
+				MongoDatabase database = client.getDatabase("stores");			
+				MongoCollection<Document> collection = database.getCollection("orders");
+				
+				DeleteResult deletedOrders = collection.deleteMany(and(eq("status","cancelled"),lt("orderPlaced", thresholdDate)));
+				System.out.println("Number of Orders removed: "+deletedOrders.getDeletedCount());
+				return deletedOrders.getDeletedCount();
```
Comment (shinyou21) [inline RemoveData/src/RemoveData.java:35]: Here we are checking only the number of orders deleted. Meaning that we are not verifying that 5 orders with status "cancelled" are deleted, but only verifying that 5 orders are canceled. Of course it uses the eq to find those satisfy the condition, but maybe we want to check if a specific order( get id number) is deleted? (check if that specific id number is deleted) 
Just a thought

Comment (Raghav2018) [reply RemoveData/src/RemoveData.java:35]: The unit test automatically creates 5 dummy records that match a specific condition before deletion, calls remove function with the same condition and verifies if 5 dummy records were deleted. I cannot verify by id unless I hard code 5 records manually and make sure they match the condition.

Comment (shinyou21) [reply RemoveData/src/RemoveData.java:35]: ok got it

**Review by shinyou21 — COMMENTED** [review]
LGTM. 
@md3822 please review the same

**Pull request merged**